CHANGELOG.md

```diff
@@ -4,8 +4,17 @@ parent: README
 nav_order: 1
 ---# Changelog
 
+## [2.1.5] - 2021-01-21
+- Add support for renaming projects
```

Comment:
This may be overpromising things. I'd add `(only allowed for projects with no powered on services)` here